terraform/modules/bq_executor_alert_policy/variables.tf

```diff
@@ -0,0 +1,56 @@
+variable "name" {
+  description = "name of alert policy"
+  type = string
+}
+
+variable "documentation" {
+  description = "Notification text in alert."
+  type = string
+  default = <<EOT
+# bq-executor job failed
+This policy is to alert when bq-executor job fails.
+
+## Failed query
+
+$${log.extracted_labels.query}
+
+## Error message
+
+$${log.extracted_labels.error}
+EOT
+}
+
+variable "filter" {
+  description = "filter condition on logs"
+  type = string
+  default = <<EOT
+protoPayload.methodName="google.cloud.bigquery.v2.JobService.InsertJob"
+severity=ERROR
+protoPayload.metadata.jobChange.job.jobStatus.jobState="DONE"
```

Comment:
Just to doublecheck, it seems like this should normally need 'AND' between the conditions, but I guess you tested this haha?

Reply:
Yep this is indeed tested at ANWB. I cannot really find it anymore in the documentation but I think the `AND` operator is assumed between newlines